client/sections/services/sections.service.coffee

```diff
@@ -15,14 +15,20 @@ sectionsServ = ($meteor, $state) ->
 
```

Comment:
Dlaczego nie sprawdzasz w create czy się powiodło? Jak wszędzie to wszędzie, chyba że o czymś nie pomyślałem